py/desitarget/data/targetmask.yaml

```diff
@@ -24,14 +24,14 @@ desi_mask:
     - [STD_WD,      34, "White Dwarf stars", {obsconditions: DARK|GRAY}]
     - [STD_BRIGHT,  35, "F-type standard for BRIGHT conditions",
         {obsconditions: BRIGHT}]
+    - [BADSKY,      36, "Blank sky locations that are imperfect but still useable",
+        {obsconditions: DARK|GRAY|BRIGHT|POOR|TWILIGHT12|TWILIGHT18}]
```

Comment:
I know that mask bits aren't supposed to have meaningful numbers, but was there a specific reason for moving this from bit 53 near the other BRIGHT_OBJECT bits to bit 36 near the standard star bits?  I expect our standards bits might expand somewhat and it will seem odd to have a BADSKY bit sitting in the middle of them.